NGitLab/Models/PipelineGetJobsQuery.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Runtime.Serialization;
+
+namespace NGitLab.Models
+{
+    [DataContract]
+    public class PipelineGetJobsQuery
```

Comment:
`PipelineJobQuery` would be more in line with the way we name query types

Reply:
I suggested `PipelineJobQuery` (`Job`, singular), not `PipelineJobsQuery`. See other types:
- `EpicQuery`
- `IssueQuery`
- `JobQuery`
- `MergeRequestQuery`

Reply:
Sorry, I thought that was because all of those things were only used in their respective client. Pipeline is singular because it's linked to the PipelineClient and Jobs plural because it's the jobs related to that pipeline.

I can make a quick pull-request if you want and make it singular.

Reply:
Please do